src/wasm_runtime/Cargo.toml

```diff
@@ -3,6 +3,9 @@ name = "wasm-runtime"
 version = "0.11.0"
 edition = "2021"
 
+[profile.release]
+debug = "full"
```

Comment:
what does this do and do we need it?

Reply:
I think this is an artefact of debugging, we should not have this by default, it would make the binary a lot bigger